### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by The-ProGrammer66**
Fixed English and updated copyright year

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -19,27 +19,29 @@
...
-    <string name="label_version" translatable="false">v%s (%d)</string>
+    <string name="label_version">You're running version %s (%d)</string>
 
     <string name="label_timeline_title">Timeline, milestones, and updates</string>
     <string name="label_timeline_releases">Releases</string>
 
-    <string name="message_trypophobia_warning">This is a warning to those with &lt;b&gt;Trypophobia&lt;/b&gt; that some images may cause discomfort. Do you want to continue?</string>
+    <string name="message_trypophobia_warning">This is a trigger warning to those with &lt;b&gt;Trypophobia&lt;/b&gt; that some images may cause discomfort. Do you want to continue anyway?</string>
+
+    <string name="redesign_notice">Easter Eggs has a new look!</string>
```
Comment (hushenghao) [inline app/src/main/res/values/strings.xml:41]: Where is this resource being used?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -19,27 +19,29 @@
...
+    <string name="label_timeline">OS Timeline</string>
     <string name="label_privacy_policy">Privacy</string>
     <string name="label_license">License</string>
-    <string name="label_email">Feedback</string>
-    <string name="label_star">Star</string>
+    <string name="label_email">Send Feedback</string>
+    <string name="label_star">Rate the App</string>
 
-    <string name="label_version" translatable="false">v%s (%d)</string>
+    <string name="label_version">You're running version %s (%d)</string>
```
Comment (hushenghao) [inline app/src/main/res/values/strings.xml:34]: I don't think it's necessary to say too much about the version number

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -53,23 +55,23 @@
     <string name="label_add_shortcut">Shortcut</string>
     <string name="label_donate">Donate</string>
 
-    <string name="preference_on">On</string>
-    <string name="preference_off">Off</string>
+    <string name="preference_on">Enabled</string>
+    <string name="preference_off">Disabled</string>
```
Comment (hushenghao) [inline app/src/main/res/values/strings.xml:59]: I do not want to modify these two switch related resources

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -19,27 +19,29 @@
     <string name="url_mail" translatable="false">mailto:dede.hu@qq.com</string>
     <string name="url_sponsor" translatable="false">https://github.com/hushenghao/AndroidEasterEggs/wiki/Sponsor</string>
 
-    <string name="char_separator">/</string>
+    <string name="char_separator" translatable="false">/</string>
```
Comment (hushenghao) [inline app/src/main/res/values/strings.xml:22]: Keep it translatable，Separators can be modified for different languages

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (The-ProGrammer66) [reply app/src/main/res/values/strings.xml:41]: Just in case the UI gets another redesign.

Comment (The-ProGrammer66) [reply app/src/main/res/values/strings.xml]: I just repurchased it into an error string

**Pull request merged**